src/error.rs

```diff
@@ -1155,8 +1155,25 @@ signed, unsigned, big, little, native, size, unit",
                     )
                     .unwrap();
                 }
-            },
 
+                RecordUpdateInvalidConstructor { location, name } => {
```

Comment:
As far as I can tell, there isn't actually any possible way of reaching this error with the current grammar, but I added a new error just in case. If you'd prefer panicking, or you have other ideas, I'm open to those too.